frontend-react/src/components/EditableCompare.tsx

```diff
@@ -66,43 +67,12 @@ export const EditableCompare = forwardRef(
         // the API call into this forwardRef well say if json is valid (to enable save button)
         const [isValidSyntax, setIsValidSyntax] = useState(true);
 
-        useImperativeHandle(
```

Comment:
Moved this [down](https://github.com/CDCgov/prime-reportstream/pull/7713/files#diff-9549b9587cf857fa88a11d9c5d5a2ad10d106f0846b78007cd5901d000c038b8R147) so we don't have an issue with referencing functions created underneath it in the code.  (As a VERY superficial aside, this is generally why I like using function declarations instead of function expressions since they're hoisted up to the top of the scope, but this doesn't apply to memoized callbacks since those will always be expression!)